packages/suite/src/types/suite/guide.tsx

```diff
@@ -1,7 +1,13 @@
 /* eslint-disable camelcase */
+import {
+    Page as GuidePageType,
+    Category as GuideCategoryType,
+    Node as GuideItemType,
```

Comment:
nitpick: Let's remove all the `*Type` suffixes, they convey no meaning
```suggestion
    Page as GuidePage,
    Category as GuideCategory,
    Node as GuideItem,
```

Personally, I wouldn't rename them at all but I'm ok with these new names.

Reply:
Resolved.